### PR TITLE
Metric Alerts schema 2018-03-01

### PR DESCRIPTION
**Pull request by snehithm**

### CODE REVIEW

**Comment by snehithm** [comment]
@Tiano2017  Please review and merge.

**Comment by snehithm** [comment]
@TianoMS Tagging the correct username

File: schemas/2015-01-01/deploymentTemplate.json [hunk 1]
```diff
@@ -536,7 +536,8 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.Insights.json#/resourceDefinitions/actionGroups" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.BatchAI.json#/resourceDefinitions/clusters" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.BatchAI.json#/resourceDefinitions/fileServers" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.BatchAI.json#/resourceDefinitions/jobs" }
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.BatchAI.json#/resourceDefinitions/jobs" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.Insights.json#resourceDefinitions/metricAlerts" }
```
Comment (TianoMS) [inline schemas/2015-01-01/deploymentTemplate.json]: >{ "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.Insights.json#resourceDefinitions/metricAlerts" } [](start = 18, length = 125)

can you put this to the line right below actionGroups?

File: tests/2018-03-01/Microsoft.Insights.metricAlerts.tests.json [hunk 2]
```diff
@@ -0,0 +1,108 @@
+{
+  "tests": [
```
Comment (TianoMS) [inline tests/2018-03-01/Microsoft.Insights.metricAlerts.tests.json]: >tests [](start = 3, length = 5)

can you add one positive test case?

Comment (snehithm) [reply tests/2018-03-01/Microsoft.Insights.metricAlerts.tests.json]: First one is a positive test case. It should successfully create a metric alert.
@TianoMS

File: schemas/2015-01-01/deploymentTemplate.json (hunk 1 above)
Comment (snehithm) [reply schemas/2015-01-01/deploymentTemplate.json]: Done.

**Pull request merged**